src/shared/credentials/credentialsProfileMru.ts

```diff
@@ -15,19 +14,14 @@ export class CredentialsProfileMru {
     public static readonly MAX_CREDENTIAL_MRU_SIZE = 5
 
     private static readonly configurationSettingName: string = 'recentCredentials'
-    private readonly _configuration: SettingsConfiguration
 
-    public constructor(configuration: SettingsConfiguration) {
-        this._configuration = configuration
-    }
+    public constructor(private readonly _context: vscode.ExtensionContext) {}
 
     /**
      * @description Returns the most recently used credentials names
      */
     public getMruList(): string[] {
-        const mru = this._configuration.readSetting<string[]>(CredentialsProfileMru.configurationSettingName)
-
-        return mru || []
+        return this._context.globalState.get<string[]>(CredentialsProfileMru.configurationSettingName, [])
```

Comment:
Should we rename `configurationSettingName` to something like `configurationStateName`?